Framework/Geometry/inc/MantidGeometry/Instrument.h

```diff
@@ -6,25 +6,22 @@
 #include "MantidGeometry/IDetector.h"
 #include "MantidGeometry/Instrument/CompAssembly.h"
 #include "MantidGeometry/Instrument/ObjComponent.h"
-
 #include "MantidKernel/DateAndTime.h"
+#include "MantidBeamline/Beamline.h"
```

Comment:
Consider avoiding this include and instead use a forward declaration (and store `Beamline` as `unique_ptr`).